sklearn/tests/test_pipeline.py

```diff
@@ -1687,3 +1648,11 @@ def test_feature_union_getitem_error(key):
     msg = "Only string keys are supported"
     with pytest.raises(KeyError, match=msg):
         union[key]
+
+
+def test_pipeline_param_validation():
+    """Run the param validation for `Pipeline`."""
+    model = Pipeline(
+        [("scaler", StandardScaler()), ("classifier", LogisticRegression())]
+    )
+    check_param_validation("Pipeline", model)
```

Comment:
Could we put that in ``test_common`` ? We could extend the parametrization of ``test_check_param_validation`` to not test just ``test_estimators``.